terraform/deployments/datagovuk-infrastructure/security.tf

```diff
@@ -0,0 +1,12 @@
+resource "aws_security_group_rule" "postgres_from_eks_workers" {
+  for_each = merge(data.terraform_remote_state.app_govuk_rds.outputs.sg_rds, {
```

Comment:
I'm confused as to why the for_each here. It looks like we're adding a rule for every database plus the CKAN one, rather than just for CKAN. Is that right?

Did you mean to remove the for_each and write `data.terraform_remote_state.infra_security_groups.outputs.sg_ckan_id` instead of `each.value` on line 10?